deep-learning/src/test/python/synapsemltest/dl/conftest.py

```diff
@@ -19,6 +21,14 @@
 )
 
 
+class CallbackBackend(object):
+    def run(self, fn, args=(), kwargs={}, env={}):
+        return [fn(*args, **kwargs)] * self.num_processes()
+
+    def num_processes(self):
+        return 1
+
```

Comment:
What is this class used for? The hardwired 1 makes me a little suspicious here

Reply:
This is kind of a mockup backend for local testing. It's used in teset_deep_text_classifier & test_deep_vision_classifier. I actually copied it from mlflow tests lol